api/main/java/com/StruckCroissant/GameDB/core/game/Game.java

```diff
@@ -35,6 +35,8 @@ public class Game implements DbModelObj {
 
   @NotBlank private final float rawgId;
```

Comment:
Annotation most likely not needed. New game entities won't be added so there is no point in validating them.

Reply:
Resolved in recent commit